docs/user/check-ref/kubernetes_wrapper.rst

```diff
@@ -0,0 +1,243 @@
+
+.. code-block:: python
+
+    # Get all pods with label ``application`` equal to ``zmon-worker``
+    kubernetes().pods(application='zmon-worker')
+    kubernetes().pods(application__eq='zmon-worker')
+
+
+    # Get all pods with label ``application`` **not equal to** ``zmon-worker``
+    kubernetes().pods(application__neq='zmon-worker')
+
+
+    # Get all pods with label ``application`` **any of** ``zmon-worker`` or ``zmon-agent``
+    kubernetes().pods(application__in=['zmon-worker', 'zmon-agent'])
+
+    # Get all pods with label ``application`` **not any of** ``zmon-worker`` or ``zmon-agent``
+    kubernetes().pods(application__notin=['zmon-worker', 'zmon-agent'])
+
+
+Methods of Kubernetes
+^^^^^^^^^^^^^^^^^^^^^
+
+.. py:function:: pods(name=None, phase=None, ready=None, **kwargs)
+
+        ::
+
+        Return list of Pods.
+
+        :param name: Pod name.
+        :type name: str
+
+        :param phase: Pod status phase. Valid values are: Pending, Running, Failed, Succeeded or Unknown.
+        :type phase: str
+
+        :param ready: Pod ready status. If None then all pods are returned.
+        :type ready: bool
+
+        :param **kwargs: Pod :ref:`labelSelectors` filters.
+        :type **kwargs: dict
+
+        :return: List of pods. Typical pod has "metadata", "status" and "spec".
+        :rtype: list
+
+.. py:function:: nodes(name=None, **kwargs)
+
+        ::
+
+        Return list of nodes. Namespace is ignored.
+
+        :param name: Node name.
+        :type name: str
+
+        :param **kwargs: Node :ref:`labelSelectors` filters.
+        :type **kwargs: dict
+
+        :return: List of nodes. Typical pod has "metadata", "status" and "spec".
+        :rtype: list
+
+.. py:function:: services(name=None, **kwargs)
+
+        ::
+
+        Return list of Services.
+
+        :param name: Service name.
+        :type name: str
+
+        :param **kwargs: Service :ref:`labelSelectors` filters.
+        :type **kwargs: dict
+
+        :return: List of services. Typical service has "metadata", "status" and "spec".
+        :rtype: list
+
+.. py:function:: endpoints(name=None, **kwargs)
+
+        ::
+
+        Return list of Endpoints.
+
+        :param name: Endpoint name.
+        :type name: str
+
+        :param **kwargs: Endpoint :ref:`labelSelectors` filters.
+        :type **kwargs: dict
+
+        :return: List of Endpoints. Typical Endpoint has "metadata", and "subsets".
+        :rtype: list
+
+.. py:function:: ingresses(name=None, **kwargs)
+
+        ::
+
+        Return list of Ingresses.
+
+        :param name: Ingress name.
+        :type name: str
+
+        :param **kwargs: Ingress :ref:`labelSelectors` filters.
+        :type **kwargs: dict
+
+        :return: List of Ingresses. Typical Ingress has "metadata", "spec" and "status".
+        :rtype: list
+
+.. py:function:: statefulsets(name=None, replicas=None, **kwargs)
+
+        ::
+
+        Return list of Statefulsets.
```

Comment:
Add link to https://kubernetes.io/docs/user-guide/petset/ (or statefulset if they re-fix-f*ck-it-up)